### PR TITLE
Add caasp-kvm

### PR DESCRIPTION
**Pull request by grahamhayes**

### CODE REVIEW

File: caasp-kvm/README.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+$ # (replace openSUSE_Leap_42.2 with your distro)
+$ sudo zypper ar obs://Virtualization:containers/openSUSE_Leap_42.2 Virtualization:containers
+$ sudo zypper in terraform-provider-libvirt
+```
+
+You are going to need a working libvirt daemon capable of using KVM.
+
+# Cluster configuration
+
+The deployment can by tuned uses some terraform variables. All of them
```
Comment (monstermunchkin) [inline caasp-kvm/README.md]: I guess, this should read "The deployment can be tuned using some terraform variables."

File: caasp-kvm/README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+is also removed from the libvirt pool by the `terraform destroy` command.
+This makes harder to test regressions between different builds.
+
+The image downloaded inside of the git checkout is *never ever* removed by
+terraform. It must be deleted manually by the user once it's no longer
+needed.
+
+The download is performed by a simple bash script invoked automatically by
+terraform during the `apply` stage. The script won't download the image if it's
+already available inside of the git checkout. This ensure no time has to
```
Comment (monstermunchkin) [inline caasp-kvm/README.md]: "ensures"

File: caasp-kvm/README.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+## cloud-init
+
+The project comes with two cloud-init files: one for the admin node, the other
+for the generic nodes.
+
+Feel free to edit them to suit your needs. Right now they are based on the files
+used by our QA engineers.
+
+Note well: the system is going to have a `root` and a `qa` users with password
```
Comment (monstermunchkin) [inline caasp-kvm/README.md]: "[...] going to have the users `root` and `qa`, both with password `linux`."

File: caasp-kvm/README.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+`linux`.
+
+# Cluster architecture
+
+The cluster is made by 1 admin node and the number of generic nodes chosen by
```
Comment (monstermunchkin) [inline caasp-kvm/README.md]: "made *of*"

File: caasp-kvm/README.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+of that network.
+
+# Creating the cluster
+
+Steps to perform:
+
+  * Configure the cluster the way you want (see above section).
+  * Execute: `terraform apply`
+
+At the end of the deployment you will see the IP address of the admin server.
```
Comment (monstermunchkin) [inline caasp-kvm/README.md]: You will also see the IP addresses of the workers. Perhaps that should be mentioned somewhere.

File: caasp-kvm/README.md [hunk 6]
```diff
@@ -43,7 +43,7 @@ You are going to need a working libvirt daemon capable of using KVM.
 
 # Cluster configuration
 
-The deployment can by tuned uses some terraform variables. All of them
+The deployment can by tuned using some terraform variables. All of them
```
Comment (monstermunchkin) [inline caasp-kvm/README.md]: "be" ;)

**Comment by kiall** [comment]
The README def needs some work :) But it's a good start! Merging though

**Pull request merged**